Sources/PortalSwift/Core/mpc/PortalMpc.swift

```diff
@@ -340,7 +340,49 @@ public class PortalMpc {
       }
     }
   }
+    
+    /// Uses the org and client backup shares to return the private key
+    ///  - Parameters:
+    ///    - cipherText: the cipherText of the client's backup share
+    ///    - method: The specific backup storage option.
+    ///    - orgShare: the stringified version of the organization's backup share
+    public func EjectWalletAndDiscontinueMPC(
+        cipherText: String,
+        method: BackupMethods.RawValue,
+        backupConfigs: BackupConfigs? = nil,
+        orgShare: String,
+        completion: @escaping (Result<String>) -> Void
+    ) {
+        if self.version != "v5" {
+            completion(Result (error: MpcError.recoverNoLongerSupported(message: "[PortalMpc] Recover is no longer supported for this version of MPC. Please use `version = v5`.")))
+        }
+
+        self.getBackupShare(cipherText: cipherText, method: method) { (result: Result<String>) in
+            guard result.error == nil else {
+                completion(Result (error: result.error!))
+                return
+            }
+            if let clientBackupShare = result.data{
+                // Call eject with clientBackupShare and orShare
+                let privateKey = self.mobile.MobileEjectWalletAndDiscontinueMPC(clientBackupShare, orgShare)
+                // Call API backend to set the client as ejected
+                let request = HttpRequest<String, [String: String]>(
+                  url: self.apiHost + "/api/v1/clients/eject",
+                  method: "POST",
+                  body: [:],
+                  headers: ["Authorization": "Bearer \(self.apiKey)"],
+                  requestType: HttpRequestType.CustomRequest
+                )
+                request.send { (result: Result<String>) in
+                    print("Notified portal of ejection")
```

Comment:
you need to add error checking here if the request fails to send that completion up